src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,130 @@
+package duke;
+
+import java.util.Scanner;
+
+public class Duke {
+
+    static final Task[] TASKS = new Task[101];
+    static int tasksI = 1;
+
+    public static void main(String[] args) {
+        // print intro message
+        start();
+        // run bot (decode task)
+        run();
+    }
+
+    public static void start() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+    }
+
+    public static void list() {
+        System.out.println("Here are the tasks in your list: ");
+        for (int i = 1; i < tasksI; i++) {
+            Task currTask = TASKS[i];
+            System.out.println(i + ". " + currTask.toString());
+        }
+    }
+
+    public static void mark(String word, Task task) {
```

Comment:
Try to be more specific in your naming of methods as 1-worded methods can usually seem to be quite ambiguous, making it difficult for readers to understand the purpose of your method